walkthrough/src/modules/app/walkthrough/_javascript.jade

```diff
@@ -134,7 +134,7 @@ questions.Questions--2
       answer-toggle(correct="true")
         :markdown
           ```
-          .SearchForm.isSuperUser {
+          .SearchForm .isSuperUser {
```

Comment:
You're directly styling `. isSuperUser` with this modification. This is [not allowed](https://github.com/gilbox/css-bliss#state) because it breaks module encapsulation.